gluish/common.py

```diff
@@ -50,6 +86,36 @@ def is_exe(fpath):
 
     return None
 
+
+def check(service):
```

Comment:
Check seems like a very generic name (also, just used once so far). How about something more catchy :), maybe:

```python
In [122]: def service_is_up(url):
     ...:     try:
     ...:         return requests.get(url).status_code == 200
     ...:     except:
     ...:         return False
```

As for logging, the requests module already does most of it, [probably](https://stackoverflow.com/questions/16337511/log-all-requests-from-the-python-requests-module).

Also, there are some edge cases:

```python
In [130]: service_is_up("http://www.github.com")
Out[130]: True # Fine.

In [135]: service_is_up("https://www.amazon.com")
Out[135]: False # Suspicious.
```

Try various UA on Amazon.

```python
In [136]: requests.get("https://www.amazon.com")
Out[136]: <Response [503]>
# Ah, Amazon specifically excludes requests UA (https://requestb.in/17aaf8o1?inspect)

In [137]: requests.get("https://www.amazon.com", headers={'User-Agent': 'nicebot'})
Out[137]: <Response [200]>
```

Reply:
yeah, I got inspired by the which method ;)